### ISSUE TITLE
Error CRSF Tocken en subscripción a newsletter

### ISSUE THREAD

**Issue by aaloy** [python-spain/symposion#2]
http://localhost:8000/newsletter/suscribe/

Añadir el token al formulario de subscripción.

**Comment by achamorro-dev** [comment]
Cuidado con los clones que el repo no esta actualizado. Ademas el token no
lo meti aposta hasta que no probaramos :-)

Voy a ver si soluciono el problema del repo de heroku y os aviso.

**Comment by aaloy** [comment]
No sé cómo funciona Heroku en profundidad, la verdad es que para los proyectos que hacemos sale mucho más barato un servidor dedicado y administrado (o varios de ellos). Me preocupa el tema de poder tener un entorno de preproducción donde poder probar los desarrollos separado del entorno de producción.

**Comment by achamorro-dev** [comment]
Por favor, no hagáis clones hasta que os diga.

**Comment by astrojuanlu** [comment]
Todo esto tiene una explicación, es un poco rollo de contar pero ya está todo en orden. El último commit de Alberto a master directamente (8646cbfa5b6a35e7b0e9711a0b0b71eacff8ffa6) ha sido para no liar más. A partir de ahora, ramas :)

**Comment by achamorro-dev** [comment]
Sí! Ahora si que tenéis la última versión. Mando correo para confirmar todo: flujo de trabajo, "personal" :)
